src/AndcultureCode.CSharp.Sitefinity.Conductors/Domain/DynamicContentRepositoryConductor.cs

```diff
@@ -48,6 +53,21 @@ ILogger<IDynamicContentRepositoryConductor> logger
             _logger = logger;
         }
 
+        public IResult<DynamicContent> Create<T>(T item) where T : SitefinityContent => Do<DynamicContent>.Try((r) =>
+        {
+            var type = typeof(T);
+            var properties = type.GetProperties(BindingFlags.Public | BindingFlags.Instance | BindingFlags.DeclaredOnly);
+
+            var dataItem = item.SetDataItemProperties(properties, DefaultProviderName);
+
+            // TODO: shouldn't this be done in the constructor?
```

Comment:
Minor: Remove or resolve `TODO`. In this case, I don't think it should be done in the constructor as it's possible the `DefaultProviderName` is changed by the user prior to the call but after the instantiation of the conductor, in order to retrieve from a different provider.